src/NUnitFramework/tests/WorkItemTests.cs

```diff
@@ -29,7 +29,7 @@ public void CreateWorkItems()
         [TearDown]
         public void DisposeWorkItems()
         {
-            _workItem.Dispose();
+            _workItem?.Dispose();
```

Comment:
Not needed as the NUnit.Analyzer suppresses this because it knows `_workItem` was initialized in the `SetUp` method